### PR TITLE
config: Allow env override of jmxfetch check_period

### PR DESCRIPTION
**Pull request by masci**
### What does this PR do?

Continuation of https://github.com/DataDog/datadog-agent/pull/3432

### Motivation

See [original PR](https://github.com/DataDog/datadog-agent/pull/3432)

### CODE REVIEW

**Review by truthbk — APPROVED** [review]
This looks great to me now! :shipit:

File: pkg/config/config.go [hunk 1]
```diff
@@ -312,6 +313,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("jmx_thread_pool_size", 3)
 	config.BindEnvAndSetDefault("jmx_reconnection_thread_pool_size", 3)
 	config.BindEnvAndSetDefault("jmx_collection_timeout", 60)
+	config.BindEnvAndSetDefault("jmx_check_period", defaults.DefaultCheckInterval/time.Millisecond)
```
Comment (albertvaka) [inline pkg/config/config.go]: Document the setting in `config_template.yaml`?

Comment (masci) [reply pkg/config/config.go]: Good catch, I merged too fast! Going to address it in a new PR

Comment (truthbk) [reply pkg/config/config.go]: My bad, addressed here: https://github.com/DataDog/datadog-agent/pull/3478

**Pull request merged**